llama_stack/distribution/datatypes.py

```diff
@@ -202,6 +202,11 @@ class StackRunConfig(BaseModel):
         description="Configuration for the HTTP(S) server",
     )
 
+    external_providers_dir: Optional[str] = Field(
```

Comment:
so this makes it work for "run" but what about "build"? should there be an extension point for building all the code that is necessary to run the external providers?

Reply:
You're right, only "run" is covered today. If we agree with the approach I can expand it to build. What do you think?

Reply:
OK sounds good.

Reply:
Can we add in the description that provider code / deps must have been built already? Otherwise it will not work.